djstripe/models.py

```diff
@@ -273,6 +273,10 @@ def has_any_active_subscription(self):
         return len(self._get_valid_subscriptions()) != 0
 
     @property
+    def active_subscriptions(self):
+        return self.subscriptions.filter(status="active", current_period_end__gt=timezone.now())
+
+    @property
     def valid_subscriptions(self):
         return self.subscriptions.exclude(status="canceled")
```

Comment:
"""Returns this cusotmer's valid subscriptions (subscriptions that aren't cancelled."""